Type 'unknown' does not satisfy the constraint 'Dictionary | undefined'.

### Which package is this bug report for? If unsure which one to select, leave blank

@crawlee/playwright (PlaywrightCrawler)

### Issue description

1. Install `crawlee@3.13.5` without playwright (so using puppeteer).
2. See this typescript error:
```
node_modules/@crawlee/playwright/internals/adaptive-playwright-crawler.d.ts:80:4 - error TS2344: Type 'unknown' does not satisfy the constraint 'Dictionary | undefined'.

80 }, PlaywrightGotoOptions> {
```



### Code sample

```TypeScript

```

### Package version

3.13.5

### Node.js version

22

### Operating system

MacOS

### Apify platform

- [ ] Tick me if you encountered this issue on the Apify platform

### I have tested this on the `next` release

_No response_

### Other context

Doesn't happen with `3.13.2`

--- a/packages/playwright-crawler/src/internals/playwright-crawler.ts
+++ b/packages/playwright-crawler/src/internals/playwright-crawler.ts
@@ -23,7 +23,7 @@ export interface PlaywrightCrawlingContext<UserData extends Dictionary = Diction
         PlaywrightContextUtils {}
 export interface PlaywrightHook extends BrowserHook<PlaywrightCrawlingContext, PlaywrightGotoOptions> {}
 export interface PlaywrightRequestHandler extends BrowserRequestHandler<LoadedContext<PlaywrightCrawlingContext>> {}
-export type PlaywrightGotoOptions = Parameters<Page['goto']>[1];
+export type PlaywrightGotoOptions = Dictionary & Parameters<Page['goto']>[1];
 
 export interface PlaywrightCrawlerOptions
     extends BrowserCrawlerOptions<PlaywrightCrawlingContext, { browserPlugins: [PlaywrightPlugin] }> {

--- a/renovate.json
+++ b/renovate.json
@@ -24,5 +24,5 @@
 		}
 	],
 	"schedule": ["every weekday"],
-	"ignoreDeps": ["crawlee", "cheerio", "playwright", "yarn"]
+	"ignoreDeps": ["crawlee", "cheerio", "playwright"]
 }

